config/hsc/processCcd.py

```diff
@@ -19,6 +19,7 @@
 config.calibrate.astrometry.matcher.maxRotationDeg=1.145916
 config.calibrate.astrometry.wcsFitter.numRejIter=3
 
+config.charImage.astrometry.wcsFitter.order = 3
 config.calibrate.astrometry.wcsFitter.order = 3
```

Comment:
I think all the astrometry settings for both should be the same, so I recommend using a loop over `(config.charImage, config.calibrate)` and modifying the values in the loop.